readthedocs/projects/version_handling.py

```diff
@@ -43,23 +43,23 @@ def comparable_version(version_string):
     """
     Can be used as ``key`` argument to ``sorted``.
 
-    The ``LATEST`` version shall always beat other versions in comparison.
-    ``STABLE`` should be listed second. If we cannot figure out the version
+    The ``STABLE`` version shall always beat other versions in comparison.
+    ``LATEST`` should be listed second. If we cannot figure out the version
```

Comment:
I don't think we want to make this change here (in this function) since it will affect more places than just the `sitemap_xml` view.